system/util/dummy_diag_publisher/CMakeLists.txt

```diff
@@ -1,69 +1,32 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(dummy_diag_publisher)
 
-add_compile_options(-std=c++14)
+### Compile options
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wno-unused-parameter -Wall -Wextra -Wpedantic)
+endif()
 
-find_package(catkin REQUIRED COMPONENTS
-  diagnostic_updater
-  dynamic_reconfigure
-  roscpp
-)
-
-find_package(fmt REQUIRED)
+### Dependencies
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
 
-generate_dynamic_reconfigure_options(
-  cfg/DummyDiagPublisher.cfg
-)
+### Target executable
+set(DUMMY_DIAG_PUBLISHER_SRC
+  src/dummy_diag_publisher_node/dummy_diag_publisher_node.cpp)
 
-catkin_package(
-  INCLUDE_DIRS
-    include
-  CATKIN_DEPENDS
-    diagnostic_updater
-    dynamic_reconfigure
-)
+set(DUMMY_DIAG_PUBLISHER_HEADERS
```

Comment:
Plese remove this since this is not used anymore

Reply:
Done in d869b6f